src/buckets/components/createBucketForm/MeasurementSchemaSection.tsx

```diff
@@ -0,0 +1,111 @@
+import React, {FC} from 'react'
+
+import {
+  AlignItems,
+  Button,
+  ComponentColor,
+  ComponentSize,
+  FlexBox,
+  FlexDirection,
+  IconFont,
+  Panel,
+} from '@influxdata/clockface'
+
+import 'src/buckets/components/createBucketForm/MeasurementSchema.scss'
+import {CLOUD} from 'src/shared/constants'
+
+let MeasurementSchemaList = null,
+  MeasurementSchema = null
+
+if (CLOUD) {
+  MeasurementSchema = require('src/client/generatedRoutes').MeasurementSchema
+  MeasurementSchemaList = require('src/client/generatedRoutes')
+    .MeasurementSchemaList
+}
+
+import {downloadTextFile} from 'src/shared/utils/download'
+
+interface Props {
+  measurementSchemaList?: typeof MeasurementSchemaList
+  isEditing?: boolean
+}
+interface PanelProps {
+  measurementSchema: typeof MeasurementSchema
+  index?: number
+}
+
+export const EditingPanel: FC<PanelProps> = ({measurementSchema, index}) => {
+  const handleDownloadSchema = () => {
+    const {name} = measurementSchema
+    const contents = JSON.stringify(measurementSchema.columns)
+    downloadTextFile(contents, name || 'schema', '.json')
```

Comment:
i think we should probably measure the usage of this new feature with some events. what do you think?